cfgov/cfgov/settings/base.py

```diff
@@ -439,7 +439,7 @@
 LOGIN_FAIL_TIME_PERIOD = os.environ.get("LOGIN_FAIL_TIME_PERIOD", 120 * 60)
 # number of failed attempts
 LOGIN_FAILS_ALLOWED = os.environ.get("LOGIN_FAILS_ALLOWED", 5)
-LOGIN_REDIRECT_URL = "/login/welcome/"
+LOGIN_REDIRECT_URL = "/admin/"
 LOGIN_URL = "/login/"
```

Comment:
Are these two settings even needed anymore?

Reply:
Yes, not least because we refer to them in our custom [`check_permissions`](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/v1/views/__init__.py#L129) view. These differ from the [Django default values](https://docs.djangoproject.com/en/3.0/ref/settings/#login-redirect-url) for these settings.